monoio-http-client/Cargo.toml

```diff
@@ -29,6 +29,7 @@ webpki-roots = { version = "0.23", optional = true }
 native-tls = { version = "0.2", optional = true }
 
 [dev-dependencies]
+monoio-http-client = { path = ".", features = ["logging"]}
```

Comment:
why we have to depend on ourself?

Reply:
I was just looking to enable logging in the example test cases so I can see the logs. I can go ahead and remove this